ledger-tool/src/bigtable.rs

```diff
@@ -439,7 +439,7 @@ pub fn bigtable_process_command(ledger_path: &Path, matches: &ArgMatches<'_>) {
         }
         ("delete-slots", Some(arg_matches)) => {
             let slots = values_t_or_exit!(arg_matches, "slots", Slot);
-            let dry_run = !value_t_or_exit!(arg_matches, "force", bool);
```

Comment:
no way to go past this `..._or_exit!()` given this:

https://github.com/solana-labs/solana/blob/d6d2840c9442420831a692391109b3c243b1d968/ledger-tool/src/bigtable.rs#L292

Reply:
Great catch

Reply:
Sorry about that lack of testing